react/react_intro/lab/lab.md

```diff
@@ -19,21 +19,21 @@ Your task today is to make a **personal website** front page. You should have a
 
 First, however, you should make sure that your React DOM renders something. Make your React DOM render a `p` tag with the text `hello world!`. Try to do this by inserting HTML **directly into a `ReactDOM.render` method**. See if you can get your JSX to render directly!
 
-Okay, so obviously, even though we *can* render JSX directly, components are still useful - at the very least for separating out information and keeping ourselves organized. Let's create a file called `Home.js` in our `src` folder. Make sure to import React, call your component `Home`, and see if you can render your name in an `h1` tag.
+Okay, so obviously, even though we *can* render JSX directly, components are still useful - at the very least for separating out information and keeping ourselves organized. Let's create a file called `Home.js` in our `src` folder. Make sure to import React, call your component function `Home`, and see if you can render your name in an `h1` tag.
```

Comment:
I didn't realize this project was here! Nice find! However we have another lab for this lesson that is being set up by another contributor. Also, the fellows just created personal sites at the end of M2 so there may be personal site fatigue. For these reasons, I suggest that we just delete this file.

Reply:
There should probably be a cleanup of this repo to handle all unused files/folders, but I will handle removing this one :)